bazel/grpc_build_system.bzl

```diff
@@ -168,7 +168,7 @@ def ios_cc_test(
             deps = ios_test_deps,
         )
 
-def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data = [], uses_polling = True, language = "C++", size = "medium", timeout = None, tags = [], exec_compatible_with = [], exec_properties = {}, shard_count = None, flaky = None):
+def grpc_cc_test(name, srcs = [], deps = [], external_deps = [], args = [], data = [], uses_polling = True, language = "C++", size = "medium", timeout = None, tags = [], exec_compatible_with = [], exec_properties = {}, shard_count = None, flaky = None, rbe_linux_only_uses_blackhole_ipv6_address = False):
```

Comment:
I feel that the `rbe_linux_only_uses_blackhole_ipv6_address` argument is a very adhoc option.
I'm not a fan of making the build system (which is already complex) even more complex to satisfy a very specific need like this one. How many tests will use this option and what does this option enable?
My feeling is that it would need to be extremely valuable to outweigh the increase of complexity in build system and test infrastructure associated with it.

Reply:
This basically addresses [this comment](https://github.com/grpc/grpc/blob/master/test/cpp/client/destroy_grpclb_channel_with_active_connect_stress_test.cc#L55) in a test.

For back though, there's an internal customer with a production workload that reported a crash. The crash was only reproduceable if using "grpclb", and the load balancer address failed to connect i.e. experienced packet loss and timed out. So I added https://github.com/grpc/grpc/blob/master/test/cpp/client/destroy_grpclb_channel_with_active_connect_stress_test.cc, which reproduced the crash (before the fix) 100% on my local workstation and in other environments, but unfortunately not on linux RBE because of the nature of how the network stack is configured for RBE containers. This change is meant to clean up that specific test, and get it to work in a tightly controlled manner (i.e. in such a way that we can guarantee that it always exercises a grpclb packet loss scenario). I could see this general kind of thing could come up again though.